collections/testtscripts.js

```diff
@@ -128,6 +128,22 @@ Meteor.methods({
         }
       });
     }
-    Meteor.call('updateTicketStatus', ticket);
+    Meteor.call('updateTicketStatus', ticket, failReason);
+  },
+
+  reOpenAndCommentTicket: function(id, failReason) {
+    var user = Meteor.user();
+    if (!user) {
+      throw new Meteor.Error(401, "You need to login to post test results");
+    }
+
+    var testscript = Testscripts.findOne(id);
+    var ticket = Tickets.findOne({ jiraId: testscript.ticketJiraId });
+
+    if (Meteor.isServer) {
+      Jira.reOpenTicket(ticket, failReason);
```

Comment:
I think as far as names go it would make sense to also call `Jira.commentTicket` here, and make the changes I will outline below in another comment.